### PR TITLE
fix: #239 sendFile accepts buffer and File

### PR DESCRIPTION
**Pull request by mahboubii**

### CODE REVIEW

**Review by jaapbakker88 — APPROVED** [review]
LGTM

File: test/test.js [hunk 1]
```diff
@@ -2901,6 +2901,12 @@ describe('Chat', () => {
 				expect(data.file).to.be.not.empty;
 			});
 
+			it('Upload a buffer', async () => {
```
Comment (ferhatelmas) [inline test/test.js:2904]: why didn't you add the other test in stream-js?

https://github.com/GetStream/stream-js/pull/389/files#diff-b3be5f00e4539d897e017e0426fb7606R22

**Review by ferhatelmas — APPROVED** [review]
LGTM

File: test/test.js (hunk 1 above)
Comment (mahboubii) [reply test/test.js:2914]: pushed a similar test 👍

**Pull request merged**